src/api/todos.ts

```diff
@@ -0,0 +1,26 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const deleteTodo = (todoId: number) => client
+  .delete(`/todos/${todoId}`);
+
+export const createTodo = ({ title, userId, completed }: Omit<Todo, 'id'>) => {
+  return client.post('/todos/', { title, userId, completed });
```

Comment:
```suggestion
  return client.post<Todo>('/todos/', { title, userId, completed });
```
And not use type casting in the function you use this `createTodo`:
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/86923089/3fe12256-d005-401f-a0ab-15a14d4231cd)

Just FYI